# Stardard touch output from MIRO's sensors

string id
string intensity
float32 duration
float32 last_touch
bool status 